### PR TITLE
Move combobox to creation page.

### PR DESCRIPTION
**Pull request by Lama64**

### CODE REVIEW

**Comment by Lama64** [comment]
Wait for merge of #354

**Comment by marcelfloe** [comment]
closes #322 
#339 should be ready to merge after merging

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (marcelfloe) [inline server/src/main/java/de/uftos/services/StudentGroupService.java:150]:
```suggestion
    for (Grade grade : allGrades) {
```

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (marcelfloe) [inline server/src/main/java/de/uftos/services/StudentGroupService.java:118]:
```suggestion
    for (Grade grade : allGrades) {
```

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (marcelfloe) [inline server/src/main/java/de/uftos/services/StudentGroupService.java:117]: What is this loop for?
Shouldn't there be no grade containing a reference to the new student group?

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (marcelfloe) [inline server/src/main/java/de/uftos/services/StudentGroupService.java:149]: I guess these loops are required to remove old grade references and set the new references?

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (Lama64) [reply server/src/main/java/de/uftos/services/StudentGroupService.java]: Yeah i don't know why i copied it over to create, it is obviously not needed.

File: server/src/main/java/de/uftos/services/StudentGroupService.java
Comment (Lama64) [reply server/src/main/java/de/uftos/services/StudentGroupService.java:145]: Yeah otherwise you get duplicate grades in the student group and duplicate student groups in the grade.
You could compare the grades of the student group in the db and the gradeRequestDto, but i feel like it doesn't make a big difference.

**Review by danieldietzler — APPROVED** [review]

**Pull request merged**
